konrad/aerosol.py

```diff
@@ -0,0 +1,264 @@
+                        #actually wanted format'
+                        self.optical_thickness_due_to_aerosol_sw= \
+                                get_aerosol_waveband_data_array(result[0].values.T, units='dimensionless', numlevels=self.numlevels, sw=True)
+                        self.single_scattering_albedo_aerosol_sw = \
+                                get_aerosol_waveband_data_array(0, units='dimensionless', numlevels=self.numlevels, sw=True) 
+                        if not self.includeAbsorption:
+                            raise ValueError('For aerosols scattering and absorption can not both be deactivated')
+                    except (ValueError):
+                        exit('Please choose valid input data, exclusion of absorption and scattering is not supported.')
+                        
+                if not self.includeAbsorption: 
+                    '''only scattering
+                        SSA'=1, Omega'=Omega,Ext'=Ext*SSA'''
+                    try:
+                        result= np.multiply(self.optical_thickness_due_to_aerosol_sw,
+                                            self.single_scattering_albedo_aerosol_sw)
+                        #the input for the aerosol waveband array has to be transposed in this case, since the array passed is already in the 
+                        #wanted format, passing the wanted format to the get aerosol waveband data array function will return the transpose of 
+                        #actually wanted format'
+                        self.optical_thickness_due_to_aerosol_sw= \
+                                get_aerosol_waveband_data_array(result[0].values.T, units='dimensionless', numlevels=self.numlevels, sw=True)
+                        self.single_scattering_albedo_aerosol_sw = \
+                                get_aerosol_waveband_data_array(1, units='dimensionless', numlevels=self.numlevels, sw=True)
+                       
+                        if not self.includeScattering:
+                            raise ValueError('For aerosols scattering and absorption can not both be deactivated')
+                    except (ValueError):
+                        exit('Please choose valid input data.')
+                    
+
+    def calculateHeightLevels(self, atmosphere):
+        '''Used to translate the aerosol forcing data given as a function of height 
+        to aerosol forcing data as a function of pressure levels
+        '''
+        heights = ty.pressure2height(atmosphere['plev'], atmosphere['T'][0, :])/1000
+        return heights
+
+
+class NoAerosol(Aerosol):
+    '''
+    No volcanic aerosol
+    '''
+    def __init__(self,atmNumlevels):
+        super().__init__(atmNumlevels,aerosol_type='no_aerosol')
+        
+    def update_aerosols(self, time, atmosphere):
+        return
+    
+    def calculateHeightLevels(self,atmosphere):
+        return
```

Comment:
You can remove the function ``calculateHeightLevels`` both in the ``NoAerosol`` class and the parent ``Aerosol`` class because for these classes it is not used. You need to keep ``update_aerosols`` because this one is called outside (somewhere in the radiation).